frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-install.tsx

```diff
@@ -0,0 +1,365 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { match } from 'react-router';
+import { safeDump } from 'js-yaml';
+import { Alert, Button, ActionGroup } from '@patternfly/react-core';
+import { ButtonBar } from '@console/internal/components/utils/button-bar';
+import { history } from '@console/internal/components/utils/router';
+import {
+  referenceForModel,
+  K8sResourceKind,
+  K8sKind,
+  k8sGet,
+  k8sCreate,
+  K8sResourceKindReference,
+  Status,
+  k8sPatch,
+  getNodeRoles,
+} from '@console/internal/module/k8s';
+import {
+  BreadCrumbs,
+  RequestSizeInput,
+  SelectorInput,
+} from '@console/internal/components/utils/index';
+import { ListPage } from '@console/internal/components/factory';
+import { NodeModel, ClusterServiceVersionModel } from '@console/internal/models';
+import { ClusterServiceVersionKind } from '@console/internal/components/operator-lifecycle-manager/index';
+import { CreateYAML } from '@console/internal/components/create-yaml';
+import { Tooltip } from '@console/internal/components/utils/tooltip';
+import { OCSServiceModel } from '../../models';
+import { NodeList } from './node-list';
+import { OCSStorageClassDropdown } from './storage-class-dropdown';
+import {
+  minSelectedNode,
+  taintObj,
+  labelObj,
+  ocsRequestData,
+} from '../../constants/ocs-install';
+
+import './ocs-install.scss';
+
+export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = React.memo((props) => {
+  const title = 'Create New OCS Service';
+  const dropdownUnits = {
+    TiB: 'TiB',
```

Comment:
Not much of a dropdown here if there is only 1 option.  I know this handles decimals but are you sure you would not like at least GiB?

Reply:
We are no longer going to show capacity field. Hence, will remove.